### PR TITLE
fix(interface): import bootstrap and font awesome properly

### PR DESCRIPTION
**Pull request by endes0**

### CODE REVIEW

File: src/interface/styles/vendor/main.scss
Comment (oxcabe) [inline src/interface/styles/vendor/main.scss:1]: Remove commented import statements and add newline at the end of the file.

File: src/interface/styles/base/variables.scss
Comment (oxcabe) [inline src/interface/styles/base/variables.scss:1]: Remove pixel based sizing comments. If we need to go back to these values, we have the commit history to recover them.

**Review by oxcabe — CHANGES_REQUESTED** [review]
A personal recommendation: configure your text-editor / IDE to add newlines at the end of files. Running Prettier also corrects to this formatting style. Maybe it's a good idea to have pre-commit hooks that run project wise linters and formatters.

Feel free to share your thoughts on this and correct the aforementioned. Once it's done, LGTM and it's approved :+1:

File: src/interface/styles/vendor/main.scss
Comment (endes0) [reply src/interface/styles/vendor/main.scss:1]: fixed

File: src/interface/styles/base/variables.scss
Comment (endes0) [reply src/interface/styles/base/variables.scss:1]: fixed

**Review by oxcabe — APPROVED** [review]
LGTM.

**Pull request merged**